conf/application-json-logger.xml

```diff
@@ -5,4 +5,8 @@
         <encoder class="uk.gov.hmrc.play.logging.JsonEncoder"/>
     </appender>
 
```

Comment:
You need to include logger package name as well I think ?
something like ``` <logger name="uk.gov" level="WARN"/>```. Its looks like you only define the log level.